tox.ini

```diff
@@ -27,19 +30,33 @@ commands =
   black -v --check --diff {toxinidir}/plugins {toxinidir}/tests
 
 [testenv:black]
+depends =
+  flynt, isort
 deps =
-  {[testenv:black_check]deps}
+  black >=23.0, <24.0
+commands =
+  black -v {[common]format_dirs}
 
+[testenv:isort]
+deps =
+  isort
 commands =
-  black -v {posargs:{toxinidir}/plugins {toxinidir}/tests}
+  isort --profile black {[common]format_dirs}
+
+[testenv:flynt]
+deps =
+  flynt
+commands =
+  flynt {[common]format_dirs}
 
 [testenv:linters]
 deps =
-  yamllint
+  {[testenv:black]deps}
+  {[testenv:isort]deps}
   flake8
-  {[testenv:black_check]deps}
-
+  yamllint
 commands =
-  {[testenv:black_check]commands}
+  black -v --check {toxinidir}/plugins {toxinidir}/tests
```

Comment:
```suggestion
  flynt {[common]format_dirs}
  black -v --check {toxinidir}/plugins {toxinidir}/tests
```

Reply:
If we want to use flynt we should add it to the linters in https://github.com/ansible-collections/cloud-content-handbook/tree/main/CI. For this PR, we should merge this as is. We can look at adding flynt later.